client/src/stores/DataStore.ts

```diff
@@ -13,8 +13,21 @@ class DataStore {
 
     const res = await axios.get("/api/subjects");
     this.subjectsLoading = false;
-    this.subjects = res.data;
+    console.log(res.data.error);
```

Comment:
Logs are great for coding phase but maybe we should not commit them.
Or we can do something even cooler, we can create our own log-service that actually logs only if we are on the dev env.

Reply:
sounds cool! most of the logs are still helping me through the rest of my tasks that's why they are still there.